test/DebuggerCommon/failedasm.js.dbg.baseline

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
you can use debuggercommon\emptyJson.dbg.baseline (which was added for this purpose only)